### PR TITLE
Net controller check

### PR DESCRIPTION
**Pull request by johanes2115**

### CODE REVIEW

File: docs/unified-test-documentation/dasharo-compatibility/359-net-controller-after-coldboot-warmboot-reboot-suspend.md [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+    ```
+
+## NET002.001 NET controller after warmboot (Ubuntu 22.04)
+
+**Test description**
+
+This test aims to verify that the network controller works and the platform
+is able to connect to the network after warmboot.
+
+**Test steps**
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/359-net-controller-after-coldboot-warmboot-reboot-suspend.md:88]: Mssing section

File: docs/unified-test-documentation/dasharo-compatibility/359-net-controller-after-coldboot-warmboot-reboot-suspend.md [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+
+1. Command from points 8. and 12. should return information indicating if the
+    network interface works correctly (`up`) or not (`down`). Example output:
+
+    ```
+    up
+    ```
+
+## NET003.001 NET controller after reboot (Ubuntu 22.04)
+
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/359-net-controller-after-coldboot-warmboot-reboot-suspend.md:146]: missing section

File: docs/unified-test-documentation/dasharo-compatibility/359-net-controller-after-coldboot-warmboot-reboot-suspend.md [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+    ```
+    ip a
+    ```
+
+1. Determine te name of the ethernet controller. It typically is `ethX` or
+    `enpXsY`, where `X` and `Y`  are numbers.
+1. Then, execute following command to check if the network interface is working
+    correctly:
+
+    ```
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/359-net-controller-after-coldboot-warmboot-reboot-suspend.md:171]: All commands in these tests are bash commands, so these sections should look as follows:

```bash
command
```

File: docs/unified-test-documentation/dasharo-compatibility/359-net-controller-after-coldboot-warmboot-reboot-suspend.md [hunk 4]
```diff
@@ -0,0 +1,275 @@
...
+    link/ether d4:93:90:16:92:f8 brd ff:ff:ff:ff:ff:ff
+    inet 192.168.4.85/24 brd 192.168.4.255 scope global dynamic noprefixroute enp46s0
+       valid_lft 40276sec preferred_lft 40276sec
+    inet6 fe80::8317:79e3:81ec:a1f4/64 scope link noprefixroute
+       valid_lft forever preferred_lft forever
+    3: wlp0s20f3: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc noqueue state DOWN group default qlen 1000
+    link/ether 14:18:c3:7d:77:06 brd ff:ff:ff:ff:ff:ff
+    ```
+
+1. Command from points 8. and 12. should return information indicating if the
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/359-net-controller-after-coldboot-warmboot-reboot-suspend.md:202]: This section should we rewritten - we should look only on interfaces `enp` and `eth`. And all that have been active before cold boot / warm boot / system reboot / suspend should be active after performing the related operation

File: docs/variants/novacustom_ns5x_tgl/test-matrix.md [hunk 5]
```diff
@@ -52,6 +52,8 @@ subjected before the release of the new binary.
 | 41.  | [Sign of life][SOL]                                    | SOL           | All                                  |
 | 42.  | [ME disable/neuter support][MNE]                       | MNE           | MNE001.001                           |
 | 43.  | [USB-C docking station SD card reader][DSD]            | DSD           | All                                  |
+| 44.  | [BIOS menu function keys][BMF]                         | BMF           | All                                  |
```
Comment (sulewskiprzemyslaw) [inline docs/variants/novacustom_ns5x_tgl/test-matrix.md:55]: This PR should concern only on net interface

**Review by sulewskiprzemyslaw — CHANGES_REQUESTED** [review]

File: docs/unified-test-documentation/dasharo-compatibility/359-net-controller-after-coldboot-warmboot-reboot-suspend.md [hunk 6]
```diff
@@ -0,0 +1,275 @@
+# Dasharo Compatibility: NET controller after coldboot/warmboot/reboot/suspend
```
[the remark this answers is not in the document]
Comment (johanes2115) [reply docs/unified-test-documentation/dasharo-compatibility/359-net-controller-after-coldboot-warmboot-reboot-suspend.md]: I changed the test module for Dasharo Stability.

File: docs/unified-test-documentation/dasharo-compatibility/359-net-controller-after-coldboot-warmboot-reboot-suspend.md [hunk 7]
```diff
@@ -0,0 +1,275 @@
...
+1. The `Thunderbolt docking station` connected to the Thunderbolt port.
+
+## NET001.001 NET controller after coldboot (Ubuntu 22.04)
+
+**Test description**
+
+This test aims to verify that the network controller works and the platform
+is able to connect to the network after coldboot.
+
+**Test steps**
```
[the remark this answers is not in the document]
Comment (johanes2115) [reply docs/unified-test-documentation/dasharo-compatibility/359-net-controller-after-coldboot-warmboot-reboot-suspend.md]: I added missing section.

File: docs/unified-test-documentation/dasharo-compatibility/359-net-controller-after-coldboot-warmboot-reboot-suspend.md (hunk 1 above)
Comment (johanes2115) [reply docs/unified-test-documentation/dasharo-compatibility/359-net-controller-after-coldboot-warmboot-reboot-suspend.md]: I added missing section.

File: docs/unified-test-documentation/dasharo-compatibility/359-net-controller-after-coldboot-warmboot-reboot-suspend.md (hunk 2 above)
Comment (johanes2115) [reply docs/unified-test-documentation/dasharo-compatibility/359-net-controller-after-coldboot-warmboot-reboot-suspend.md]: I added missing section.

File: docs/unified-test-documentation/dasharo-compatibility/359-net-controller-after-coldboot-warmboot-reboot-suspend.md (hunk 3 above)
Comment (johanes2115) [reply docs/unified-test-documentation/dasharo-compatibility/359-net-controller-after-coldboot-warmboot-reboot-suspend.md]: I corrected the commands.

File: docs/unified-test-documentation/dasharo-compatibility/359-net-controller-after-coldboot-warmboot-reboot-suspend.md (hunk 4 above)
Comment (johanes2115) [reply docs/unified-test-documentation/dasharo-compatibility/359-net-controller-after-coldboot-warmboot-reboot-suspend.md]: I corrected the section.

File: docs/variants/novacustom_ns5x_tgl/test-matrix.md (hunk 5 above)
Comment (sulewskiprzemyslaw) [reply docs/variants/novacustom_ns5x_tgl/test-matrix.md:55]: @johanes2115 and how about that?

Comment (johanes2115) [reply docs/variants/novacustom_ns5x_tgl/test-matrix.md:55]: Nie są to moje zmiany, przesuwałem w inne miejsce w dokumencie linijkę z pod spodu, prawdopodobniee git zczytał to jako zmiana.

**Review by macpijan — APPROVED** [review]

**Pull request merged**